http-clients/src/main/java/org/triplea/http/client/web/socket/WebSocketConnection.java

```diff
@@ -49,17 +57,21 @@
       onMethod_ = {@VisibleForTesting})
   private boolean connectionIsOpen = false;
 
+  @Setter(
+      value = AccessLevel.PACKAGE,
+      onMethod_ = {@VisibleForTesting})
+  private HttpClient httpClient = HttpClient.newHttpClient();
+
   private final URI serverUri;
 
   private boolean closed = false;
 
+  private WebSocket client;
+
   @Getter(
       value = AccessLevel.PACKAGE,
       onMethod_ = {@VisibleForTesting})
-  @Setter(
-      value = AccessLevel.PACKAGE,
-      onMethod_ = {@VisibleForTesting})
-  private WebSocketClient client;
+  private final WebSocket.Listener webSocketListener = new WebSocketListener();
```

Comment:
These two variables are named a bit too similarly, it's a bit difficult to keep them straight:
```
  private WebSocketConnectionListener listener;
  private final WebSocket.Listener webSocketListener = new WebSocketListener();
```

Can they be renamed reasonably or restructured perhaps?

Reply:
I renamed them to make the difference a little bit clearer.